### ISSUE TITLE
KeyError: ‘render_modes’ with Gym == 0.25.0

### ISSUE THREAD

**Issue by ntoxeg** [Bam4d/Griddly#212]
**Describe the bug**
Trying to create an environment with `gym.make` results in an exception
```
File "/Users/ntoxeg/Library/Caches/pypoetry/virtualenvs/narca-622vlgQS-py3.10/lib/python3.10/site-packages/gym/envs/registration.py", line 625, in make
    render_modes = env_creator.metadata["render_modes"]
KeyError: ‘render_modes'
```

**To Reproduce**
Install latest Gym (0.25.0 in my case).
Invoke something like
```
env = gym.make(
        "GDY-Drunk-Dwarf-v0",
        player_observer_type=gd.ObserverType.VECTOR,
        level=0,
   )
```

**Expected behavior**
Getting an instance of an environment.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 12.5

**Comment by ntoxeg** [comment]
I don’t know why Poetry installed the newest version even though I can see it pinned in the `requirements.txt` but I guess this report can be useful for squashing the rest of the bugs with newer Gym.

**Comment by Bam4d** [comment]
relates to #196

**Comment by Bam4d** [comment]
will be fixed in #282
